src/cascadia/TerminalApp/Commandline.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "Commandline.h"
+
+using namespace TerminalApp;
+
+Commandline::~Commandline()

```

Comment:
Do you need to declare this guy or can it just be default?

Reply:
(and if it's default, did you rule of 5 it and if not, do you even need to declare it at all?)

Reply:
not anymore! good eye